commands/user_test.go

```diff
@@ -52,3 +52,76 @@ func (s *MmctlUnitTestSuite) TestSearchUserCmd() {
 		s.Require().Equal("Unable to find user 'example@example.com'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestListUserCmdF() {
+	s.Run("Listing users with paging", func() {
+		emailArg := "example@example.com"
+		mockUser := model.User{Username: "ExampleUser", Email: emailArg}
+		page := 0
+		perPage := 1
+
+		cmd := &cobra.Command{}
+		cmd.Flags().Int("per-page", perPage, "")
+		cmd.Flags().Int("page", page, "")
+
+		printer.Clean()
```

Comment:
just one last nit: for clarity, would you be open to move the `printer.Clean()` calls to be the first thing done at the beginning of each test case, just right after the `s.Run`?

I think they can be missed if they are in the middle of the test rather than at first.